<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChooseExportType</class>
 <widget class="QDialog" name="DialogChooseExportType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>选择数据导出方案</string>
  </property>
  <widget class="QPushButton" name="pushButtonAll">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>124</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>导出当前文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCurrentPage">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>124</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>导出当前页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>选择数据导出类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>勾选时,导出文件将强制使用UTF-8编码保存导出文件
不勾选时,默认使用原数据文件相同的编码保存导出文件</string>
   </property>
   <property name="text">
    <string>使用UTF-8编码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
